flow/activities/flowable.go

```diff
@@ -350,22 +353,34 @@ func (a *FlowableActivity) SyncFlow(
 		}
 		logger.Info("no records to push")
 
-		err := dstConn.ReplayTableSchemaDeltas(ctx, flowName, recordBatch.SchemaDeltas)
+		dstConnForSchemaChange, connErr := connectors.GetCDCSyncConnector(ctx, config.Destination)
+		if connErr != nil {
+			return nil, fmt.Errorf("failed to get destination connector for schema changes: %w", connErr)
+		}
+		defer connectors.CloseConnector(ctx, dstConnForSchemaChange)
+		err := dstConnForSchemaChange.ReplayTableSchemaDeltas(ctx, flowName, recordBatch.SchemaDeltas)
```

Comment:
nice thing here is with future PR we could move ReplayTableSchemaDeltas into its own interface, get _that_ connector instead here, & then skip when ErrUnsupportedFunctionality